website/templates/issue.html

```diff
@@ -96,6 +127,32 @@
                target="_blank"
                rel="noopener noreferrer">{{ object.url }}</a>
         </p>
+        <div>
+            {% if object.get_label_display == "Trademark Squatting" %}
+                <div class="domain-infrig-info">
+                    <div>
+                        <h3>Infringing Domain Name</h3>
+                        <p>{{ infringing_domain_name }}</p>
+                    </div>
+                    <div>
+                        <h3>Infringing Domain URL</h3>
+                        <p>{{ infringing_domain_url }}</p>
+                    </div>
+                    <div>
+                        <h3>Registration No</h3>
+                        <p>{{ registration_number }}</p>
+                    </div>
+                    <div>
+                        <h3>Serial No</h3>
```

Comment:
```suggestion
                        <h3>Serial Number</h3>
```

Reply:
resolved in https://github.com/OWASP-BLT/BLT/pull/2655/commits/3e76e5b033074d878415807aa0742ac40a9e7996